src/main/java/educationalproject/programmingstuff/servicies/dto/UserResponseDto.java

```diff
@@ -1,11 +1,14 @@
 package educationalproject.programmingstuff.servicies.dto;
 
 import educationalproject.programmingstuff.model.Order;
-import lombok.Value;
+import lombok.Getter;
+import lombok.Setter;
 
 import java.util.List;
 
-@Value
+//@Value //Lombock's annotation incompatible with Jackson. Got hassle in tests.
```

Comment:
Strange, we used it on my previous project and had no problems

Reply:
@Data is a good alternative

Reply:
@Data is ok. I've check